developersociety/django-template
# Improve 404 page

Sources
https://app.forecast.it/project/P-57/scoping/T6537

Description
- Changes the template for 404 pages to inherit from the base template
- Updates the language to be more user friendly
- Includes a link back to the homepage (this might have to be updated if doing a multilingual site but this is a start!)

Setup instructions
-  Pull down this branch
-  Install cookiecutter with `pip install --user cookiecutter` if you don't have it already
-  Deactivate the virtual environment and navigate to the Sites directory
-  Use cookiecutter to create a new project from this template using `cookiecutter ~/Sites/django-template` - if this doesn't work and it says cookiecutter isn't installed, try `python -m cookiecutter ~/Sites/django-template/`
-  Navigate into that new project and create a virtual environment for it with mkvirtualenv yourprojectname
- `make reset`

How to test
- Run your test project with `./manage.py runserver`
- Go to http://127.0.0.1:8000/404 and check how the page looks. If this 404 test page isn't found and shows the generic Python not found message, you need to edit `/project/urls.py` (this is another small issue that I intend to fix with a different PR!) and move the 'Wagtail catch-all' bit down to the bottom of the file.

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-b6d646b1d6e54011ada8169aee543231.
